instrumentation/opentelemetry-instrumentation-flask/src/opentelemetry/instrumentation/flask/__init__.py

```diff
@@ -366,12 +386,18 @@ def _wrapped_before_request(
     request_hook=None,
     tracer=None,
     excluded_urls=None,
+    excluded_paths=None,
     enable_commenter=True,
     commenter_options=None,
 ):
     def _before_request():
-        if excluded_urls and excluded_urls.url_disabled(flask.request.url):
+        if (
```

Comment:
Would be worthwhile to factor this into an independently testable function along with some of the logic on line [330](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1869/files#diff-2c109e76591442fd2a0e5ac717f0148877d9e99cbc76619e4876942c9c86543dR330)?